conf/application.conf

```diff
@@ -111,5 +112,15 @@ microservice {
       auth-token = "localToken"
       environment = "localEnvironment"
     }
+
+    hip {
+      url = "http://localhost:9156"
+      auth-token = "localToken"
+      environment = "localEnvironment"
```

Comment:
I think environment can be deleted, it doesn't seem to serve a purpose unless you know of what it's for?